validation/citrus-validation-json/src/test/java/com/consol/citrus/integration/JsonPathSegmentVariableExtractorIT.java

```diff
@@ -0,0 +1,12 @@
+package com.consol.citrus.integration;
+
+import com.consol.citrus.annotations.CitrusXmlTest;
+import com.consol.citrus.testng.spring.TestNGCitrusSpringSupport;
+import org.testng.annotations.Test;
+
+public class JsonPathSegmentVariableExtractorIT extends TestNGCitrusSpringSupport {
+
+    @Test
+    @CitrusXmlTest
```

Comment:
I just had an idea.. could we have the `@Ignore` of the "no empty test methods" sonar rule on the `@CitrusXmlTest` annotation? :D